geo_fence_operations/test_views.py

```diff
@@ -157,3 +157,83 @@ def test_valid_payload(self):
         )
         self.assertEqual(response.json()["message"], "Geofence Declaration submitted")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+
+class GeoZonePostTests(APITestCase):
```

Comment:
I was thinking to keep ed269 suffix to make things clear , as blender will implement new spec some time later.